src/Microsoft.TestPlatform.Common/ExtensionFramework/Utilities/TestExtensions.cs

```diff
@@ -307,6 +375,27 @@ internal TestExtensions GetExtensionsDiscoveredFromAssembly(string extensionAsse
         }
     }
 
+    /// <summary>
+    /// Gets the cached extensions for the current process.
+    /// </summary>
+    /// 
+    /// <returns>A map representing the cached extensions for the current process.</returns>
+    internal IDictionary<string, ISet<string>> GetCachedExtensions()
```

Comment:
I don't know what's the team preference but I usually prefer to be as specific as possible on return (even more when "private" api) as it might avoid doing cast or reconstruction of a type later on.

Reply:
Yeah if this is a Dictionary of hashset let's just use those types.

Reply:
Done.